java.java

```diff
@@ -0,0 +1,11 @@
+// package descrition
+import java.io.*;
+
+/** 
+ * Docummentation for java class
+ * multiline docummentation goes ere
+ */
+class HelloWorldApp {
+    /* multiline coment goes ere
```

Comment:
Typo inside comment in token 'coment'. Possible corrections:cement (63),comment (51),moment (44)